<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="text_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>161</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border :1px solid ;
border-top-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>161</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border :1px solid ;
border-top-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
